029-IoTEdge/Student/Challenge-01.md

```diff
@@ -1,6 +1,6 @@
 # Challenge 1: Deploy IoTHub/Edge
 
-**[Home](../README.md)** - [Next Challenge >](./Challenge-02.md)
+[Next Challenge >](./Challenge-02.md)
 
 ## Requirements to Begin 
```

Comment:
This whole "Requirements to Begin" section for the pre-requisites should be its own "Challenge 0".  I would move this section out and add a new Challenge-00.md using the challenge template, but make it focused on all of the pre-requisites.  "Challenge 0 - Prerequisites" is our common naming pattern in WTH for this type of stuff.

Reply:
> This whole "Requirements to Begin" section for the pre-requisites should be its own "Challenge 0". I would move this section out and add a new Challenge-00.md using the challenge template, but make it focused on all of the pre-requisites. "Challenge 0 - Prerequisites" is our common naming pattern in WTH for this type of stuff.

There was already a discussion with @gfilicetti about this. I thought this was already resolved??

Reply:
The only comments I see above from Gino are regarding a "Pre-Requisites" header on Challenge 2?   I'm talking about the giant block at the top of Challenge 1.  I think given the length of things here, this really should follow the pattern of being a Challenge 0 like in our other hacks.